### ISSUE TITLE
Feature Request! Full control of Blog Archive Page layout.

### ISSUE THREAD

**Issue by Flipback8** [pojome/elementor#5024]
I would like to be able to visually control every element of the Blog Archive page. The module that you can drag into the page to show the archive page does not allow for individual post modification. I recently tried to make this design and it was impossible with Elementor Pro. 
Blog Archive Page Design: http://prntscr.com/k517ev

## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by bainternet** [comment]
Duplicate of #4440 

Thanks,
Please follow guidelines for opening a new feature request, so we can help you better.
This report is missing key information: 

* **Prerequisites**.
* Description and examples.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce the new Loop Builder released with Elementor Pro v3.8.0! 🥳 

With the new Loop Grid widget, you can customize your post listings with custom templates! 💪 
Drag & Drop widgets to create complex designs, fine-tune your Loop Grid layout, add alternate templates, and much more!

This is just the first phase. More customization is coming!

Check out the article here: https://elementor.com/help/loop-grid/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
